CVEs/CVE-2021-23364.json

```diff
@@ -0,0 +1,23 @@
+{
+    "CVE": "CVE-2021-23364",
+    "state": "PUBLISHED",
+    "repository": "https://github.com/browserslist/browserslist.git",
+    "prePatch": {
+        "commit": "c091916910dfe0b5fd61caad96083c6709b02d98",
```

Comment:
I think it's better to have the `prePatch` be the parent commit from the `postPatch`. 
That way the `prePatch` and `postPatch` commits are closer to each other. 

```suggestion
        "commit": "209adf9e0051fa39a2b25354cffd493300f34b02",
```

Reply:
TBH, I got confused when I want to decide which one is the `prePatch` of the `postPatch`, so your correction is helpful for me. so, thanks for helping me understand about it @erik-krogh